src/Chirp.Razor/wwwroot/css/style.css

```diff
@@ -777,4 +787,18 @@ ul.nav li a:hover {
   padding: 5px 10px;
   border-radius: 15px;
   font-size: 14px;
+}
+
+.modeButton {
+  padding: 20px;          
```

Comment:
could make position fixed. Would eliminate the wasted space that was mentioned by Alex.
`
.modeButton {
  position: fixed;
  left: 20px;
  top: 20px;
}
`

Reply:
either solution is fine

Reply:
Nice, I did exactly that (and made it right-aligned). Thanks for the suggestion! d22944f003c67394f0f155c973a4f7eb95c2c0ab

Reply:
@alexop1000 are you happy with this? Otherwise I'll just add it to the nav bar.

Reply:
Seems a bit out of place, but if you guys are fine with it we can keep it. However, the button should probably say "Theme" at the least.

Reply:
I think you're right. I added it to the end of nav bar, hopefully everybody is cool with this. e24f20b218db707f7b4e8de35935c9eebafc64c5